masonry/src/widget/flex.rs

```diff
@@ -1178,9 +1178,9 @@ impl Widget for Flex {
 
     fn paint(&mut self, ctx: &mut PaintCtx, scene: &mut Scene) {
         // paint the baseline if we're debugging layout
-        if ctx.debug_paint && ctx.widget_state.baseline_offset != 0.0 {
+        if ctx.debug_paint() && ctx.baseline_offset() != 0.0 {
```

Comment:
I'm fine with method getters (`ctx.debug_paint()`) here, but IIRC Olivier had arguments against it (i.e. abstraction on a simple property)?

Reply:
The reason I chose to make `debug_paint` a getter rather than just making the attribute public, was to disallow mutation of the value.

Reply:
Yeah I get the motivation, that's why I think it's fine for me, also to serve as documentation for external custom widgets (which can't use these properties directly).